markdownlint.js

```diff
@@ -182,9 +212,11 @@ program
   .version(pkg.version)
   .description(pkg.description)
   .usage('[options] <files|directories|globs>')
+  .arguments('[options] <files|directories|globs>')
   .option('-f, --fix', 'fix basic errors (does not work with STDIN)')
   .option('-s, --stdin', 'read from STDIN (does not work with files)')
-  .option('-o, --output [outputFile]', 'write issues to file (no console)')
+  .option('-o, --output <outputFile>', 'write issues to file (no console)')
```

Comment:
Thanks for fixing the brackets to show required value - mind doing the other options, too?

Reply:
Updated all of them